napari/layers/labels/_tests/test_labels_mouse_bindings.py

```diff
@@ -0,0 +1,101 @@
+import numpy as np
+import collections
+from napari.layers import Labels
+from napari.utils.interactions import (
+    ReadOnlyWrapper,
+    mouse_press_callbacks,
+    mouse_move_callbacks,
+    mouse_release_callbacks,
+)
+
+
+def test_paint():
+    """Test painting labels with different brush sizes."""
+    data = np.ones((20, 20))
+    layer = Labels(data)
+    layer.brush_size = 10
+    layer.mode = 'paint'
+    layer.selected_label = 3
+    layer._last_cursor_coord = (0, 0)
```

Comment:
Should this be deleted now that we are using `last_cursor_coord` that is local to `paint()`?

Reply:
yes, dropped